www/styles/login.css

```diff
@@ -0,0 +1,45 @@
+.logo_smartquote{
+  width: 92%;
+  margin: auto;
+  padding: 23px;
+  margin-top: 40%;
+  margin-left: 4%;
+  margin-right: 50%;
+}
+
+.backgroundInitialPage{
+  background-image: url("../assets/images/background.png"); 
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+
+.user_information_box{
+  margin-left: 15%;
+  margin-right: 20%;
+  border-width: 5px;
+  border-style: inset;
+  border-color: rgba(204, 204, 204, 0) rgba(204, 204, 204, 0) #FF5907 rgba(204, 204, 204, 0);
+  background-color: rgba(204, 204, 204, 0);
+  color: #444;
+}
+
+.button_login{
+  margin-right: 50%;
+  margin-left: 39%;
+  margin-bottom: 28px;
+  background-color: #FF5907;
+  color: white;
+  padding: 3px 11px 3px 11px;
+  font-size:20px;
+  border-style: hidden;
+}
+
+.button_sing_in{
+  text-align: center;
+  font-size: 15px;
+  color: #423B3B;
+  margin-right: 50%;
+  margin-left:  37%;
+}
```

Comment:
1. Style naming are fine. But we need a convention to avoid this messing up. Sorry it was not defined before. I see you used `button_login` and `backgroundInitialPage` and also Johnny and I used things like `button-primary`. There should be guideline foreach of those 3 conventions. But For now, let's keep everything as `button-primary` class.